src/Subscription.php

```diff
@@ -664,14 +664,14 @@ public function modifiers()
     }
 
     /**
-     * Get a modifier instance by name.
+     * Get a modifier instance by ID.
      *
      * @param  int  $id
      * @return \Laravel\Paddle\Modifier|null
      */
     public function modifier($id)
     {
-        return $this->modifiers->find($id);
```

Comment:
Won't this create an N+1 problem like [here previously](https://github.com/laravel/cashier-paddle/pull/83)?